modules/game_interface/widgets/statsbar.lua

```diff
@@ -479,7 +479,7 @@ function StatsBar.reloadCurrentTab()
     if statsBarsDimensions[dimension] then
         return constructStatsBar(dimension, currentStats.placement)
     else 
-        print("No stats bars dimensions found: ", dimension, " on reloadCurrentTab()")
+        -- print("No stats bars dimensions found: ", dimension, " on reloadCurrentTab()")
```

Comment:
I wonder why is this unnecessary?
I've added this because if someone is adding new stats bars it might fall into this line and help development debug.
Thinking of that, I wonder if there's a separate way to print only for debug

Reply:
![image](https://github.com/mehah/otclient/assets/114332266/4658b4f4-8e9e-4106-843b-31a319406c80)

i hide bar
logout/ login
and all those prints came up.

Reply:
Hmm it seems to be a bug, can you create an issue about this? I'll fix it.

Reply:
undo